sklego/preprocessing/repeatingbasis.py

```diff
@@ -116,6 +122,31 @@ def transform(self, X):
         """
         check_is_fitted(self, ["pipeline_"])
         return self.pipeline_.transform(X)
+      
+    def inverse_transform(self, X):
+        """Transform RBF features back to the input range. Outputs a numpy array. 
+
+        Parameters
+        ----------
+        X : 2D array-like of shape (n_samples, n_features)
+            Can be either a pandas.DataFrame or a numpy array.
+            The RBF columns to transform back must appear first.
+
+        Returns
+        -------
+        X_transformed : array-like with the reconstruction of the original values
+                        in input_range in the first column. Will be of the same type as X.
+        """
+        
+        check_is_fitted(self, ["pipeline_"])
+        
+        if isinstance(X,np.ndarray):
+          Xarr = check_array(X[:,:self.n_periods], estimator=self, ensure_2d=True)
```

Comment:
I would be more comfortable with something along the following lines:

If line 144 becomes 

```py
Xarr = check_array(X, estimator=self, ensure_2d=True, ensure_min_features=self.n_periods)[:, :self.n_periods]
```

which also convert to array (some) dataframe-like objects and maybe it's even possible to avoid checking for array instance.